.packit.yaml

```diff
@@ -9,15 +9,15 @@ jobs:
 - job: propose_downstream
   trigger: release
   metadata:
-    dist_git_branch: master
+    dist-git-branch: master
 - job: propose_downstream
   trigger: release
   metadata:
-    dist_git_branch: f30
+    dist-git-branch: f30
 - job: propose_downstream
   trigger: release
   metadata:
-    dist_git_branch: f29
+    dist-git-branch: f29
```

Comment:
[f29 has reached EOL](https://www.linuxtechnews.com/fedora-29-officially-reached-end-of-life-eol-on-november-26th-2019/), I'd suggest using the `fedora-all` alias.

Reply:
@tantale ^